internal/bcdb/ledger_query_procesor_test.go

```diff
@@ -370,7 +370,6 @@ func createDataUpdatesFromBlock(block *types.Block) map[string]*worldstate.DBUpd
 }
 
 func TestGetBlock(t *testing.T) {
-	t.Parallel()
```

Comment:
Why did you remove this?

Reply:
It causes the `interenal/bcdb` unit tests to throw an OOM... too many tests together. Even if I run just the BCDB tests, not the entire suite...